docs/pages/frameworks/svelte.md

```diff
@@ -57,10 +57,19 @@ If you'd rather not use the CDN for assets, you can create a build task that cop
 One caveat is there's currently Svelte only supports `bind:value` directive in `<input>`, `<textarea>` and `<select>`, but you can still achieve two-way binding manually.
 
 ```jsx
-// This doesn't work
+// ❌ This doesn't work
 <sl-input bind:value="name"></sl-input>
-// This works, but it's a bit longer
-<sl-input value={name} oninput={event => message = event.target.value}></sl-input>
+<sl-select bind:value="job">
+  <sl-option value="designer">Designer</sl-option>
+  <sl-option value="developer">Developer</sl-option>
+</sl-select>
+
+// ✅ This is a bit longer, but it works
+<sl-input value={name} oninput={event => name = event.target.value}></sl-input>
+<sl-select value={job} onsl-change={event => job = event.target.value}>
```

Comment:
Using `sl-input` will work the same way here. Also adding some whitespace for clarity.

```suggestion

<sl-select value={job} onsl-input={event => job = event.target.value}>
```